e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -337,9 +337,6 @@
               "Logging:LogLevel:Microsoft": {
                 "value": "Error"
               },
-              "TEST_BUILD_NUMBER": {
-                "value": "<Test.BuildNumber>"
-              },
```

Comment:
removed from the TRC environment variables and passed instead through the yaml testInfo field